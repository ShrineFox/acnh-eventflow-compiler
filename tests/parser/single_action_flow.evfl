flow Test():
    TestActor2.Action3(4, 'z', 3.7, false)
