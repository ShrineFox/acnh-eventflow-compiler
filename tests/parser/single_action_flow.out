RootNode[name=Test, vardefs=[], out_edges=[Event0]]
ActionNode[name=Event0, action=Action3(param0, param1, param2, param3), params={'param0': TypedValue(type=Type(type='int'), value=4), 'param1': TypedValue(type=Type(type='str'), value="'z'"), 'param2': TypedValue(type=Type(type='float'), value=3.7), 'param3': TypedValue(type=Type(type='bool'), value=False)}, out_edges=[]]

